src/js/todo.js

```diff
@@ -1,4 +1,8 @@
+/* eslint-disable import/no-unresolved */
 /* eslint-dsibale no-loop-func, no-func-assign, no-class-assign */
+import deleteItem from './deletetodo.js';
```

Comment:
- [ ] Hey team good job so far, however, your tests are failing please make sure to fix the issue.

![Screenshot from 2022-11-09 21-33-47](https://user-images.githubusercontent.com/89228743/200951924-f6d2bdac-07ee-412a-86c0-b77d84221969.png)

Reply:
Okay thank you, for the screenshot, you can please submit for a second review.

Reply:
@ridwanediallo i dont know on the dashboard it looks like it already been approved? I am so confused

Reply:
Oh okay, when was it approved?

Reply:
Right after you said require change
Take a look at the exercise 
![image](https://user-images.githubusercontent.com/83097009/201025010-70e5ab5c-b9ef-4ae8-880b-49e2edaa7a36.jpeg)

Reply:
@ridwanediallo